.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--- Provide a general summary of your changes in the Title above -->
```

Comment:
Suggesting an alternative PR Template:
```suggestion
#### Description
<!--- Brief description of changes made and purpose of PR, include link to Issue Number --->

#### Changes
<!--- Bullet pointed list of main changes made, no need to repeat every single file name or every single commit but a general idea to guide the reader --->

#### Additional Notes
<!--- OPTIONAL: notes about new documentation added, testing to consider, impact on other areas of code or work if relevant --->
```

Reply:
thanks, i like the additional notes section. reworking to include Simon's feedback and yours and will push for review.

Reply:
Hi @sl-slaing @MattCline-SL @pdaulbyscottlogic, incorporated feedback from the discussions we had and pushed a new version, please let me know if i have missed anything. Thanks!

Reply:
I'd have issue # as its own section